source/cloud/azure/single-node.md

```diff
@@ -0,0 +1,99 @@
+# Azure Single Node RAPIDS
+
+## Create Azure Virtual Machine with GPU, Nvidia Driver and Nvidia Container Runtime
+
+Nvidia maintains an image that pre-installs Nvidia drivers and container runtimes,
+we recommend using [this image](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/nvidia.ngc_azure_17_11?tab=Overview) as the starting point.
+
+### Create VM Via Azure Portal
```

Comment:
Perhaps to make this clear that it's an either/or we could show them as different options?

```suggestion
### Option 1: Create VM Via Azure Portal
```

Reply:
I hope we can use a plugin like cython's doc: https://cython.readthedocs.io/en/latest/src/tutorial/cython_tutorial.html. User can switch between the two tabs (and synced through the rest of the doc).

Reply:
The current blocker is that it doesn't support dark themes that well. If I can find time can probably hack around it pretty easily, for now implementing the suggestion above.

Reply:
That would be great! Could you open an issue to track this going forwards?

Reply:
https://github.com/rapidsai/deployment/issues/44